File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int val=INT_MIN,temp=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==0){
                val=max(val,temp);
                temp=0;
            }
            else
                temp++;
        }
        return max(temp,val);

    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public static int findMaxConsecutiveOnes(int[] nums) {
        int count=0 ,temp=-1;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=1){
                if(temp<count) {temp=count;}
                count=0;
                continue;
            }
            count++;
            }
        return Math.max(count,temp);
    }
}